app/views/application_letters/check.html.erb

```diff
@@ -25,7 +23,11 @@
   <tr>
     <th><%= ApplicationLetter.human_attribute_name(:motivation) %>:</th>
     <td id="application-motivation" data-orig-text="<%= @application_letter.motivation %>">
-      <%= truncate @application_letter.motivation, length: 300 do link_to I18n.t('events.list.read_on'), 'javascript:jQuery("#application-motivation").text(jQuery("#application-motivation").attr("data-orig-text"))' end %>
+      <%= truncate @application_letter.motivation, length: 300 do
+        link_to I18n.t('events.list.read_on'),
+          'javascript:jQuery("#application-motivation")
+            .text(jQuery("#application-motivation").attr("data-orig-text"))'
```

Comment:
The second dot is called on a different jQuery object if you look closely (within the .text() call). If you think the line is still too long, you could indeed move it to the next line, but then also indent it once more to make clear that the functions are not both targeting the same object.